mlflow/spark.py

```diff
@@ -163,7 +165,6 @@ def save_model(spark_model, path, mlflow_model=Model(), conda_env=None, jars=Non
     >>> model = ...
     >>> mlflow.spark.save_model(model, "spark-model")
     """
-    dfs_tmpdir = dfs_tmpdir if dfs_tmpdir is not None else DFS_TMP
```

Comment:
The reason to use this pattern is so that a user can pass a `dfs_tmpdir` which is allowed to be `None`. Otherwise, when invoking this API, there is no way to provide `dfs_tmpdir` and ask for the default (e.g., `dfs_tmpdir=sys.environ['MY_DFS_TMPDIR']`)